homeassistant/components/cover/mqtt.py

```diff
@@ -99,6 +142,18 @@ def async_added_to_hass(self):
         This method is a coroutine.
         """
         @callback
+        def tilt_updated(topic, payload, qos):
+            """The tilt was updated."""
+            if (payload.isnumeric() and
+                    self._tilt_min <= int(payload)
+                    and int(payload) <= self._tilt_max):
```

Comment:
Use the min/max comparison structure shown in `message_received` below, so `self._tilt_min <= int(payload) <= self._tilt_max`. (Yay Python!)